testsuite/performance/tests/src/main/java/org/keycloak/performance/dataset/Creatable.java

```diff
@@ -41,23 +42,26 @@ public default boolean createCheckingForConflict(Keycloak adminClient) {
             } else {
                 String responseBody = response.readEntity(String.class);
                 response.close();
-                if (response.getStatus() == 409) { // some endpoints dont't throw exception on 409, throwing here
-                    throw new ClientErrorException(HTTP_409_SUFFIX, response);
-                }
-                if (responseBody != null && !responseBody.isEmpty()) {
-                    logger().trace(responseBody);
-                    setRepresentation(EntityTemplate.OBJECT_MAPPER.readValue(responseBody, (Class<REP>) getRepresentation().getClass()));
-                } else {
-                    setId(getCreatedId(response));
+                switch (response.getStatus()) {
+                    case 201: // created
+                        if (responseBody != null && !responseBody.isEmpty()) {
+                            logger().trace(String.format("Response status: %s, body: %s", response.getStatus(), responseBody));
+                            setRepresentation(EntityTemplate.OBJECT_MAPPER.readValue(responseBody, (Class<REP>) getRepresentation().getClass()));
+                        } else {
+                            setId(getCreatedId(response));
+                        }
+                        break;
+                    case 409: // some client endpoints dont't throw exception on 409 response, throwing from here
+                        throw new ClientErrorException(HTTP_409_SUFFIX, response);
+                    default:
+                        throw new RuntimeException(String.format("Error when creating entity %s.", this), new WebApplicationException(response));
                 }
             }
         } catch (ClientErrorException ex) {
             if (ex.getResponse().getStatus() == 409) {
                 conflict = true;
-                logger().trace("entity already exists");
+                logger().debug(String.format("Entity %s already exists.", this));
```

Comment:
`logger().debug("Entity %s already exists.", this));`

is a bit easier version of this?